tools/docusaurus-to-antora-conversion-scripts/convert-docs.sh

```
@@ -0,0 +1,87 @@
+
+  local content="$(cat "$markdown_file")"
+
+  local output_file_dir="$(dirname "$output_file")"
+  mkdir -p "$output_file_dir"
+
+  # Extract the content of the meta description tag
+  local description="$(echo "$content" | sed -n 's/.*<meta name="description" content="\([^"]*\)".*/\1/p')"
+
+  # Remove the head element from the source Markdown file and save it
+  local cleaned_content="$(echo "$content" | sed '/<head>/,/<\/head>/d')"
+  local cleaned_file="$(mktemp)"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)